mkdocs.yml

```
@@ -93,7 +93,8 @@ nav:
       - Coding Guide: developer-guide/developer.md
       - OVN-Kubernetes Container Images: developer-guide/image-build.md
       - Documentation Guide: developer-guide/documentation.md
-      - Local Testing Guide: developer-guide/testing.md
+      - Launching OVN-Kubernetes: installation/launching-ovn-kubernetes-on-kind.md
+      - Local Testing Guide: developer-guide/local_testing.md
```

```suggestion
      - Local Testing Guide: developer-guide/local_testing_guide.md
```
seems like that's what you renamed the file to